app/templates/events/volunteerInformation.html

```diff
@@ -0,0 +1,161 @@
+        <label for="phoneSelect">Phone Number</label><br>
+        <input class="displayCheckbox noprint" type="checkbox" name="selected_items" id="emailSelect" checked>
+        <label for="emailSelect">Email</label><br>
+      </div>
+      <div class="col-12 col-md">
+        <div class="row">
+          <div class="col-auto mb-2 float-right" >
+            <button class="btn btn-primary" id="tableCardToggle" nowrap>Table View</button>
+          </div>
+          <div class="col-auto align-items-end">
+            <button id="printVolunteerInfo" class="btn-primary btn">Print</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+ 
+  <hr>
+
+  <div id="volunteerInformationCardToPrint">
+    <div class="row sort-here">
+    {% set combinedParticipants = attendedUser + rsvpUser %}
+    {% set printedNames = [] %}
+    {% set uniqueParticipants = combinedParticipants | unique %}
+    {% for participant in uniqueParticipants %}
+      {% set isAttended = participant in attendedUser %}
+      {% set isRSVPed = participant in rsvpUser %}
+      {% set isWaitlist = participant in waitlistUser %}
+    
+      {% if isRSVPed %}
+        <div class="rsvpSelect col-12 col-md-6 volunteerInfoEntries print-column" data-user="{{participant.username}}">
+          <h4 class="nameSelect" nowrap><b>{{ participant.firstName }} {{ participant.lastName }}</b></h4>
+          <ul>
+            <div class="phoneSelect" nowrap>{{ participant.phoneNumber }}</div>
+            <div class="emailSelect" nowrap>{{ participant.email }}</div>
+            <div class="dietRestrictionSelect">Dietary Restrictions: {{ participant.dietRestriction }}</div>
+          </ul>
+        </div>
+      {% endif %}
+      {% if isAttended%}
+        <div class="attendedSelect col-12 col-md-6 volunteerInfoEntries print-column" data-user="{{participant.username}}">
+          <h4 class="nameSelect" nowrap><b>{{ participant.firstName }} {{ participant.lastName }}</b></h4>
+          <ul>
+            <div class="phoneSelect" nowrap>{{ participant.phoneNumber }}</div>
+            <div class="emailSelect" nowrap>{{ participant.email }}</div>
+            <div class="dietRestrictionSelect">Dietary Restrictions: {{ participant.dietRestriction }}</div>
+          </ul>
+        </div>
+      {% endif %}
+      {% if isWaitlist %}
+        <div class="waitlistSelect col-12 col-md-6 volunteerInfoEntries print-column" data-user="{{participant.username}}">
+          <h4 class="nameSelect" nowrap><b>{{ participant.firstName }} {{ participant.lastName }}</b></h4>
+          <ul>
+            <div class="phoneSelect" nowrap>{{ participant.phoneNumber }}</div>
+            <div class="emailSelect" nowrap>{{ participant.email }}</div>
+            <div class="dietRestrictionSelect">Dietary Restrictions: {{ participant.dietRestriction }}</div>
+          </ul>
+        </div>
+      {% endif %}
+    {% endfor %}
+    </div>
```

Comment:
Same as previous comment, why not just elif?

Reply:
Answered above.